llap-server/src/java/org/apache/hadoop/hive/llap/daemon/impl/QueryTracker.java

```diff
@@ -70,7 +71,7 @@ public class QueryTracker extends AbstractService {
 
   private static final Logger LOG = LoggerFactory.getLogger(QueryTracker.class);
   private static final Marker QUERY_COMPLETE_MARKER =
-      new Log4jMarker(new Log4jQueryCompleteMarker());
+      new Log4jMarker(StaticMarkerBinder.getSingleton().getMarkerFactory(), new Log4jQueryCompleteMarker());
```

Comment:
`StaticMarkerBinder.getSingleton().getMarkerFactory()` is the correct way to initialize the `Maker`?

Reply:
- This is how old versions behave. The corresponding function signature was changed 2 years ago at https://github.com/apache/logging-log4j2/pull/1032 .